components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -59,26 +59,34 @@
     // Notification channel claims.
     public static final String VERIFY_EMAIL_CLIAM = "http://wso2.org/claims/identity/verifyEmail";
     public static final String EMAIL_VERIFIED_CLAIM = "http://wso2.org/claims/identity/emailVerified";
+    public static final String EMAIL_ADDRESS_PENDING_VALUE_CLAIM =
+            "http://wso2.org/claims/identity/emailaddress.pendingValue";
```

Comment:
We have set of additional claims for emails such as emails.home, emails.work which are supported via SCIM.. shouldn't we support verification on those upon update?

Reply:
Currently, we will be providing email verification on update feature only for email address persisted in "http://wso2.org/claims/emailaddress" claim